src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -99,5 +142,26 @@ private void Snapshot()
         this.virtualMemoryUsage = this.currentProcess.PrivateMemorySize64;
         this.userProcessorTime = this.currentProcess.UserProcessorTime.TotalSeconds;
         this.privilegedProcessorTime = this.currentProcess.PrivilegedProcessorTime.TotalSeconds;
+        this.cpuUtilization = this.GetCpuUtilization();
+        this.numberOfThreads = this.currentProcess.Threads.Count;
+    }
+
+    private IEnumerable<Measurement<double>> GetCpuUtilization()
+    {
+        var userProcessorUtilization = (this.currentProcess.UserProcessorTime.TotalSeconds - this.collectionUserProcessorTime) /
```

Comment:
nit: Use `this.userProcessorTime` instead of `this.currentProcess.UserProcessorTime.TotalSeconds` as the `Snapshot` method will update `this.userProcessorTime` before `GetCpuUtilization` method is called.

Reply:
updated based on your suggestion; thank you